test/performance/odp_timer_perf.c

```diff
@@ -17,6 +17,7 @@
 
 #define MODE_SCHED_OVERH  0
 #define MODE_SET_CANCEL   1
+#define MODE_SET_EXPIRE   2
```

Comment:
Since set functions have been replaced by start functions. This could be named as e.g. START_EXPIRE or EXPIRE_RESTART. Also, the SET_CANCEL could be renamed similarly.

Reply:
v2.